cmd/controller-manager/templateHelper.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2018-2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"os"
+	"text/template"
+)
+
+type Parameters struct {
+	Params map[string]string
+}
+
+type TemplateHelper struct {
+	Parameters   Parameters
+	TemplatePath string
+	TemplateList []string
+}
+
+// Creates a new templates helper and populates the values for all
+// templates properties. Some of them (like the hostname) are set
+// by the user in the custom resource
+func NewTemplateHelper(extraParams map[string]string) *TemplateHelper {
+
+	log.Info("Installing Monitoring Resources")
+
+	params := Parameters{Params: extraParams}
+
+	templatePath := "./templates/"
+	if _, err := os.Stat(templatePath); os.IsNotExist(err) {
+		templatePath = "../templates/build/enmasse-latest/install/bundles/monitoring"
+		if _, err := os.Stat(templatePath); os.IsNotExist(err) {
+			panic("cannot find templates")
+		}
+	}
+
+	return &TemplateHelper{
+		TemplatePath: templatePath,
+		TemplateList: GetTemplateList(),
+		Parameters:   params,
+	}
+}
+
+func GetTemplateList() []string {
+	templateList := []string{
+		"010-PrometheusRules-kube-metrics.yaml",
```

Comment:
Consider getting rid of the number prefix (that is only for ordering when installing via bundles), and moving them to controller-manager/src/main/resources/templates instead. I think it belongs there now.